netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -777,6 +777,7 @@ _Requires_exclusive_lock_held_(_net_ebpf_ext_sock_addr_lock) static void _net_eb
         if (!delete_all && entry->timestamp > expiry_time) {
             break;
         }
+#pragma warning(suppress : 6001) /* list entry is non-null */
```

Comment:
nit: does this suppress the warning for the rest of the file? If yes, should we use `push/pop` to supress the warning?

Reply:
No.  "suppress" means for the following line only.  "disable" would do for the rest of the file.  See https://learn.microsoft.com/en-us/cpp/preprocessor/warning?view=msvc-170